README.md

```diff
@@ -1,3 +1,7 @@
+![docker-deploy-foxy](https://github.com/open-br/ros_ws/workflows/docker-deploy-foxy/badge.svg?branch=master)
+![docker-deploy-dashing](https://github.com/open-br/ros_ws/workflows/docker-deploy-dashing/badge.svg?branch=master)
+![docker-build](https://github.com/open-br/ros_ws/workflows/docker-build/badge.svg?branch=master)
+
```

Comment:
Nice!

Reply:
I think this is going to work only when the main branch gets the push. For now, we can check the docker-build job:

![docker-build](https://github.com/open-br/ros_ws/workflows/docker-build/badge.svg?branch=feature/add-github-action)

Reply:
@ivan1993br, it seems that the docker deployment didn't work. I suggest enabling the push trigger on a separate branch to test it